tests/python-example-auth-scram-sha-1.sh

```diff
@@ -2,6 +2,6 @@
 
 set -ex
 
-pip3 install pymongo
+pip3 install -r requirements.txt
```

Comment:
Let's avoid installing those packages globally. The easiest way to do that is probably (still) virtualenv. Maybe there is something more modern and current

Reply:
Dependabot supports [Pipenv](https://pipenv.pypa.io/) which I think is the best candidate, because we would no longer need to use pip and virtualenv separately. This seems to solve our problem.

Reply:
And we also need to create a new virtualenv using the correct Python version installed on the system hence this slight [mess](https://github.com/FerretDB/dance/pull/801/files#diff-13a8e9762c839ef76971d9a70bce544a8957326a7e9f2074b9ed63163c64bef1R7).

Reply:
Yes, not sure how I ended up installing a third-party library. I am using the correct module and running the binaries from the virtual environment.

Reply:
:pray: